tests/integration/study_data_blueprint/test_renewable.py

```diff
@@ -422,3 +463,24 @@ def test_lifecycle(
         obj = res.json()
         description = obj["description"]
         assert bad_study_id in description
+
+        # Cannot duplicate a fake cluster
+        fake_id = "fake_id"
+        res = client.post(
+            f"/v1/studies/{study_id}/areas/{area_id}/renewable/{fake_id}?new_cluster_name=duplicata",
+            headers={"Authorization": f"Bearer {user_access_token}"},
+        )
+        assert res.status_code == 404
+        obj = res.json()
+        assert obj["description"] == f"Cluster: '{fake_id}' not found"
+        assert obj["exception"] == "ClusterNotFound"
+
+        # Cannot duplicate with an existing id
+        res = client.post(
+            f"/v1/studies/{study_id}/areas/{area_id}/renewable/{other_cluster_id1}?new_cluster_name={other_cluster_id1}",
```

Comment:
Please, don't use an ID for a name.
It could be a good idea to have the same name but a different casing, for instance "OTHER CLUSTER 1" (add a comment `# note the different case`).